### DISCUSSION TITLE
C program learning BEGIN!(From a novice programmer

### DISCUSSION THREAD

**Discussion by zhangjiaming06** [community/community#144395] [Programming Help]
Hello,everyone.Here is a self-introduction and study plan of Ming.,Written out also has a certain self-encouragement nature.。

I am now studying computer science at Central South University in China, and I am an undergraduate freshman who loves technology and is eager to learn and acquire new knowledge. I will continue to update my learning blog in github to record my learning footprint. At present, the C language is selected as the entry language, and some cpp programming competitions and information security content are also repaired, and they will continue to be updated in the future!

The big goal of programming is to be able to reach the proficiency level next year, according to the C++ direction in the figure below and finish (I hope to be able to update it all the time), and network security is also to hope to reach the proficiency level in one year (there may be an element of big talk, but I believe that hard work will pay off)
<img width="931" alt="Image" src="https://github.com/user-attachments/assets/4f297471-5a49-47b0-8fe3-5ff1db93050e">
The computer curriculum at the school was difficult to accept to a certain extent, so I was in the self-learning stage.

I hope to integrate into the GitHub family in the future and make progress together with everyone！！

**Comment by tuves** [comment]
Welcome to the GitHub Community, @zhangjiaming06, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
